packages/gatsby-theme-aio/package.json

```diff
@@ -80,6 +80,7 @@
     "builtin-status-codes": "^3.0.0",
     "classnames": "^2.2.6",
     "core-js": "^3.16.3",
+    "dependency": "^0.0.1",
```

Comment:
Mhh looks weird why is it needed ?

Reply:
@timkim looks like this is failing the tests

Reply:
Oh I accidently added `dependency` and my work machine was all configured for artifactory. The latest commit should fix it now.